README.md

```diff
@@ -1,16 +1,14 @@
-Simple Decorator for Textarea
-=============================
+# Simple Decorator for Textarea
```

Comment:
actually textarea and textinput, while we're at it (original did not support textarea)